lib/compress.js

```diff
@@ -1569,6 +1577,14 @@ merge(Compressor.prototype, {
                         }
                         return node;
                     }
+                    if (node instanceof AST_Assign && node.operator == "="
+                        && node.left instanceof AST_SymbolRef
+                        && assign_as_unused) {
```

Comment:
Would you mind making assign_as_unused the first condition in the if? It's faster if assign_as_unused is not in effect, and the same speed otherwise.